lib/tasks/dev.rake

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+namespace :dev do
+  require 'factory_bot_rails'
+  require 'faker'
+
+  desc 'Generate test data'
+  task :generate_data, [:dev_num] => :environment do |_, args|
+    get_args(args)
+
+    create_programming_language
+    create_language
+
+    programming_languages = ProgrammingLanguage.all
+    languages = Language.all
+
+    @args[:dev_num].times do
+      FactoryBot.create :developer,
+                        programming_languages: programming_languages.sample(rand(1..4)),
+                        languages: languages.sample(rand(1..3))
+    end
+  end
+end
+
+def create_programming_language
+  return false if ProgrammingLanguage.find_by(name: 'ruby').present?
```

Comment:
You can use `exists?` here instead of `presents?`.
Shorter and more readable.
`ProgrammingLanguage.exists?(name: 'ruby')`
Oftentimes, it's faster. I don't know the case here though.

See more: 
https://github.com/jp-ryuji/playbook/wiki/Useful-Methods
http://semaphoreci.com/blog/2017/03/14/faster-rails-how-to-check-if-a-record-exists.html

Reply:
But I wonder why you return when there's `ruby`

Reply:
@ntrung2006 
Then you should check all of the values you want to create.
Just checking `ruby` doesn't make sense.
Anyway, take a loot at `find_or_create_by`.

Reply:
Fixed! @jp-ryuji